src/components/FormsControls.module.scss

```diff
@@ -10,7 +10,7 @@
 
 .formsControls input {
   padding: 0.5rem 1rem;
-  border: 1px solid #fff;
+  border: 1px solid #ddd;
```

Comment:
It's better to use $color-name pattern ins css.

Reply:
ok i'll fix it all in another request